### PR TITLE
Fix bugs for compute of Bias; Add support labels 0 and 1; Increase minimum out iterations

### PR DESCRIPTION
**Pull request by PetrovKP**

### CODE REVIEW

File: algorithms/kernel/svm/svm_train_kernel.h [hunk 1]
```diff
@@ -55,9 +55,9 @@ struct SVMTrainTask
 
     SVMTrainTask(size_t nVectors) : _cache(nullptr), _nVectors(nVectors) {}
 
-    Status init(algorithmFPType C, const NumericTablePtr & wTable);
+    Status init(algorithmFPType C, const NumericTablePtr & wTable, NumericTable & yTable);
```
Comment (ShvetsKS) [inline algorithms/kernel/svm/svm_train_kernel.h:58]: why not const reff?

File: algorithms/kernel/svm/svm_train_boser_impl.i [hunk 2]
```diff
@@ -486,19 +478,26 @@ SVMTrainTask<algorithmFPType, ParameterType, cpu>::~SVMTrainTask()
  * \brief Initialize the intermediate data used in SVM training
  *
  * \param[in] C     Upper bound in constraints of the quadratic optimization problem
+*  \param[in] wTable        Pointer to numeric table that contains weights
+*  \param[in] yTable        Pointer to numeric table that contains class labels
  */
 template <typename algorithmFPType, typename ParameterType, CpuType cpu>
-services::Status SVMTrainTask<algorithmFPType, ParameterType, cpu>::init(algorithmFPType C, const NumericTablePtr & wTable)
+services::Status SVMTrainTask<algorithmFPType, ParameterType, cpu>::init(algorithmFPType C, const NumericTablePtr & wTable, NumericTable & yTable)
```
Comment (ShvetsKS) [inline algorithms/kernel/svm/svm_train_boser_impl.i:485]: `ReadColumns<algorithmFPType, cpu> mtY(yTable, 0, 0, _nVectors);`
why not const reff for `yTable`

File: algorithms/kernel/svm/svm_train_kernel.h (hunk 1 above)
Comment (PetrovKP) [reply algorithms/kernel/svm/svm_train_kernel.h:58]: I will correct in the next PR.

File: algorithms/kernel/svm/svm_train_boser_impl.i (hunk 2 above)
Comment (PetrovKP) [reply algorithms/kernel/svm/svm_train_boser_impl.i:485]: I will correct in the next PR.

**Pull request merged**